jetty-core/jetty-http3/jetty-http3-client-transport/src/main/java/org/eclipse/jetty/http3/client/transport/internal/HttpReceiverOverHTTP3.java

```diff
@@ -62,7 +62,9 @@ public Content.Chunk read(boolean fillInterestIfNeeded)
         boolean last = !byteBuffer.hasRemaining() && data.isLast();
         if (last)
             responseSuccess(getHttpExchange(), null);
-        return Content.Chunk.from(byteBuffer, last, data);
+        Content.Chunk chunk = Content.Chunk.from(byteBuffer, last, data);
+        data.release();
+        return chunk;
```

Comment:
Ditto, plus it would remove the need for the local variable:
```suggestion
        return Content.Chunk.fromWithImplicitRetain(byteBuffer, last, data);
```

yeah I know the name is the difficult part!